flow/datatypes/numeric.go

```diff
@@ -113,8 +143,17 @@ func GetNumericTypeForWarehouse(typmod int32, warehouseNumeric WarehouseNumericC
 	}
 
 	precision, scale := ParseNumericTypmod(typmod)
+	if !warehouseNumeric.IsValidPrecision(precision) {
+		precision = warehouseNumeric.MaxPrecision()
+	}
+
+	if !warehouseNumeric.IsValidScale(scale) {
+		_, defaultScale := warehouseNumeric.DefaultPrecisionAndScale()
+		scale = defaultScale
```

Comment:
```suggestion
		_, scale = warehouseNumeric.DefaultPrecisionAndScale()
```
`_` is not just a throwaway identifier in Go